user/pages/03.commerce2/02.developer-guide/12.recurring/01.subscriptions-overview/docs.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Subscriptions overview
+taxonomy:
+    category: docs
+---
+
+This section of the documentation provides an overview of the Subscription entity and related entities, plugins, and objects. Specifically, this section describes:
+* Billing schedules, the billing schedule plugin, Billing periods, and the Prorater plugin
+* The Subscription type plugin and Charges
+* Recurring order types and order item types
+* The PurchasableEntitySubscriptionTrait and the Initial order
+
```

Comment:
I think the overview page could use a bit more introduction. I'd like to see us give some sample use cases rather than just listing the entities and plugins we'll be describing. For example, an introduction might say something like, "Subscriptions and recurring billing in Commerce 2.x are created and managed via the Commerce Recurring module suite. It supports a wide variety of types of subscriptions, payment plans, proration, etc. Some sample use cases include..."

And then for those sample use cases, we should pick some simple real world use cases. "A magazine subscription charged on the 5th of each month. A software subscription charged monthly starting on the date the customer purchases." Ideally these could even be shown in a table where the first column describes the use case and then additional columns identify things like prepaid vs. postpaid, rolling vs. fixed, etc. Real world use cases are great for helping people fit the concepts we describe into their existing mental model for subscriptions.

Reply:
Agreed. Even using the text from the project page would help.

Reply:
I put this in the top-level introductory page that precedes this "technical overview": user/pages/03.commerce2/02.developer-guide/12.recurring/docs.md